apps/web/src/views/Farms/Farms.tsx

```diff
@@ -217,38 +215,21 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
   )
   const archivedFarms = farmsLP
 
-  const stakedOnlyFarms = useMemo(
-    () =>
-      activeFarms.filter(
-        (farm) =>
-          farm.userData &&
-          (new BigNumber(farm.userData.stakedBalance).isGreaterThan(0) ||
-            new BigNumber(farm.userData.proxy?.stakedBalance).isGreaterThan(0)),
-      ),
-    [activeFarms],
-  )
+  const getStakedFarms = useCallback((farmsData) => {
```

Comment:
if it's a function without any dependencies, we can put it outside of the component